libraries/core_libs/network/src/tarcap/packets_handlers/vote_packet_handler.cpp

```diff
@@ -25,14 +26,15 @@ void VotePacketHandler::process(const PacketData &packet_data, const std::shared
     return;
   }
 
-  peer->markVoteAsKnown(vote_hash);
-
-  if (vote_mgr_->voteInUnverifiedMap(vote_round, vote_hash) || vote_mgr_->voteInVerifiedMap(vote)) {
```

Comment:
you cant just delete checking unverified & verified queues and replace it by cache. What if some vote is not in your cache anymore but it is in verified queue ? You would add again to unverified queue. You can check that new cache first but then you have to check also both queues as was it before